CONTRIBUTING.md

```diff
@@ -1,26 +1,12 @@
-# [wip] Contributing
+<!-- DO NOT EDIT THIS FILE! -->
+<!-- This file has been autogenerated by `poetry run pymedphys dev propagate` -->
+<!-- Please instead edit the file found at: -->
+<!--     pymedphys\lib\pymedphys\docs\contributing\index.md -->
+<!-- and then run `poetry run pymedphys dev propagate --copies` -->
 
-This document is a work in progress.
-
-## Required tools
+# Contributing
```

Comment:
There are sections that discuss environments based on OS/platform.
Do you want to reference that section here?
There are strict constraints regarding meeting the criteria of black, isort, mypy, pylint, etc. although that gets baked in when the pre-commit hook is installed.
Do you want to discuss this here or reference where it's discussed?

Reply:
Yup, this contributing file requires a lot of work. Not for this PR though if that's okay.